virtual_ship/instruments/ctd.py

```diff
@@ -71,6 +71,13 @@ def simulate_ctd(
     WINCH_SPEED = 1.0  # sink and rise speed in m/s
     DT = 10.0  # dt of CTD simulation integrator
 
+    if len(ctds) == 0:
+        print(
+            "No CTDs provided. Parcels currently crashes when providing an empty particle set, so no CTD simulation will be done and no files will be created."
+        )
+        # TODO when parcels supports it this check can be removed.
```

Comment:
OK, so this is what https://github.com/OceanParcels/parcels/issues/1629 is about? I'll try fix that asap